Test/ui/CapabilityTest.js

```diff
@@ -0,0 +1,28 @@
+const { Builder, By, until } = require("selenium-webdriver");
+const chai = require("chai");
+
+const webdriver = require("selenium-webdriver");
+
+describe("CapabilityTest", () => {
+  it("should scroll to find capability name after clicking the Employee button", async () => {
+    let driver = await new Builder().forBrowser("chrome").build();
+
+    try {
+      await driver.get("http://localhost:3000/");
+
+      await driver.findElement(By.linkText("Employee")).click();
+
+      await driver.wait(until.elementLocated(By.id("jobs-container")), 5000);
+
+      await driver.executeScript("arguments[0].scrollIntoView(true);", await driver.findElement(By.id("jobs-container")));
+
+      await driver.wait(until.elementLocated(By.css("#job-table tbody tr:first-child td:last-child")), 5000);
+
+      const capabilityName = await driver.findElement(By.css("#job-table tbody tr:first-child td:last-child")).getText();
+
```

Comment:
I guess we have picked up this double spacing from Shauns notes? haha

Reply:
> I guess we have picked up this double spacing from Shauns notes? haha

@jkelso662 It's that damn ESLinter!!!!